src/containers/ViewArticle/ViewArticle.js

```diff
@@ -129,13 +129,15 @@ class ViewArticle extends Component {
             <div className="col s2"><i className="fas fa-thumbs-down dislikeButton"></i> 3</div>
             <div className="col s1"><i className="fas fa-bookmark bookmarkButton"></i></div>
             <div className="col s1"><i className="fas fa-share-alt shareButton"></i></div>
-            {!this.props.user.isAuth && <StarRatings
-                    rating={3.03}
+            {(!this.props.user.isAuth
```

Comment:
here, I am checking if the user is logged in or if the user is the author of the article and display ratings without access to rate